ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -147,13 +149,20 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
         position={[9]}
       />
     ),
-    video: (props: ComponentToRenderProps) => (
-      <LivePageMediaPlayer
-        blocks={props.blocks}
-        className="mediaStyles"
-        css={styles.bodyMedia}
-      />
-    ),
+    video: (props: ComponentToRenderProps) =>
+      isLive() ? (
+        <LegacyMediaPlayer
+          blocks={props.blocks}
+          className="mediaStyles"
+          css={styles.bodyMedia}
+        />
+      ) : (
+        <LivePageMediaPlayer
+          blocks={props.blocks}
+          className="mediaStyles"
+          css={styles.bodyMedia}
```

Comment:
Kinda feel like these styles should be on the `LivePageMediaPlayer` component directly, rather than be passed/set as props. Just to make it very clear where they are being set from.

Reply:
@shayneahchoon Do we need to pass down the `className` and `css` props anymore since `css` is now set directly on the `LivePageMediaPlayer` component>